L1Trigger/L1TMuonEndCap/src/AngleCalculation.cc

```diff
@@ -242,6 +231,50 @@ void AngleCalculation::calculate_angles(EMTFTrack& track) const {
   if ((vstat & vmask3) != 0 || vstat == 0)
     vstat |= vmask3;
 
+
+  // FIXME: describe changes, add configuration parameter
```

Comment:
Is this for the new dTheta cleaning logic?  If so, that's great!  But as you say, it should be configurable, preferably from FW version.  I need to ask Alex if this is already deployed, or when it will be ...